src/esp/gfx/replay/Player.h

```diff
@@ -40,20 +33,11 @@ namespace replay {
  */
 class Player {
  public:
-  using LoadAndCreateRenderAssetInstanceCallback =
-      std::function<esp::scene::SceneNode*(
-          const esp::assets::AssetInfo&,
-          const esp::assets::RenderAssetInstanceCreationInfo&)>;
-  using ChangeLightSetupCallback =
-      std::function<void(const gfx::LightSetup& lights)>;
-
   /**
    * @brief Construct a Player.
    * @param callback A function to load and create a render asset instance.
    */
-  explicit Player(const LoadAndCreateRenderAssetInstanceCallback&
-                      loadAndCreateRenderAssetInstanceCallback,
-                  const ChangeLightSetupCallback& lightSetupCallback);
+  explicit Player(const PlayerCallbacks& callbacks);
```

Comment:
```suggestion
  explicit Player(PlayerCallbacks callbacks);
```
To match the other comment I made.